language-support/hs/bindings/src/DA/Ledger/Convert.hs

```diff
@@ -70,7 +70,9 @@ lowerCommands = \case
         commandsWorkflowId = unWorkflowId (fromMaybe (WorkflowId "") wid),
         commandsApplicationId = unApplicationId aid,
         commandsCommandId = unCommandId cid,
-        commandsParty = unParty party,
+        commandsParty = "", -- deprecated in favor of actAs
```

Comment:
Note that this means the Haskell bindings are not backwards compatible. This is fine since we don’t use command submisions in `daml ledger` but for example that would be an issue in party allocation since that is used in `daml ledger`.

Reply:
> This is fine since we don’t use command submissions

This means I can keep the change as is, right? I you want, I could do something like putting the head of the `actAs` list in the `party` field, and the tail in the `act_as` field.